share/spice/just_delete_me/content.handlebars

```diff
@@ -3,13 +3,8 @@
     {{notes}}
   </p>
 {{/if}}
-{{#if delete_url}}
```

Comment:
I understand your concerns, and the description now includes "Instructions:" though, maybe being more verbose and using "Account deletion instructions" might be better. To be considered.

As for the instructions not being clear, this is the fault of the provided instructions, especially in the "Plenty of Fish" case above, but IMHO the remedy to this would be to provide a PR to justdelete.me project itself, fixing the vague instructions there.

The IA for plenty of fish currently looks like this. 
![screen shot 2016-04-04 at 23 28 57](https://cloud.githubusercontent.com/assets/1518025/14263823/1c98da9e-fabd-11e5-8dd2-99bc7101c90b.png)